### DISCUSSION TITLE
1개 리스트에서 잔여백신 시도

### DISCUSSION THREAD

**Discussion by jeong0327** [kimytsc/covid-rest-vaccine-macro#59] [Q&A] [answered]

**Comment by kimytsc** [comment, marked as answer]
네이버 잔여백신 API 결과에 따르는 것이라, 1개보다는 다수의 병원이 보다 효율적입니다.

그리고 API 호출 간격에 의해 이미 잔여백신이 사라진 경우도 존재하므로 꾸준히 기다려보시면 잡힐 가능성이 높아집니다.
